_posts/2024-09-20-release 2.1-preview.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Preview LXQt Release 2.1
+slug: preview-release-2-1
+date: 2024-09-20 21:25
+promoted: true
+categories: blog
+---
+
+Here's a overview of the coming LXQt release 2.1 due in mid November as usual. The main target is the experimental **Wayland session** but also some other improvements will be included.
+
+#### Wayland Session
+
+A new package [lxqt-wayland-session](https://github.com/lxqt/lxqt-wayland-session) will be available to install - without it there won't be seen any changes, following the "modular" philosophy of LXQt: don't bother users with unwanted or unused things. If installed "Session Settings" will look like this:
+
+![Session Settings](../../../../../images/posts/waylandsettings.png)
+
+As Wayland is quite different to X11 some features are not available and some settings are handled by the compositor and not anymore by LXQt, so you may see some messages like that:
+
+![unsupported settings](../../../../../images/posts/unsupported.png)
+
+Similar to the window manager under X11 users can choose a Wayland compositor - any wlroots-compatible compositor should work, in addition to kwin-wayland. Hyprland, Labwc, kwin, Niri, Sway, River and Wayfire will be supported out of the box providing a basic configuration file for each of them. Using different screen lockers under Wayland including `swaylock`, `waylock` and `hyprlock` is supported too.
+
+At the moment there are 3 dedicated panel backends, xcb (X11), general wlroots  and kwin_wayland - using the latter many features of the panel as moving windows to workspaces, workspace switcher, urgency hints, show desktop will be enabled under Wayland. Other backends can be created and added in future, enabling similar features for other compositors like sending to scratchpad instead of minimizing windows, workspace pagers and else. PRs are welcome.
+
+As some applications will not work under Wayland (e.g. redshift, some clipboard manager exc) their auto starting can be limited to X11 only, while on the other side every compositor can start Wayland-only applications like `kanshi` or `xdg-desktop-portal-wlr`.
+
+Other fixes for Wayland have been applied overall, including the desktop window of `PCManFM-Qt` which has also been improved for multi monitor setups.
```

Comment:
There are also general fixes and workarounds for Wayland; for example, the desktop window of `PCManFM-Qt` has been improved for use with multi monitor setups.